server-adapters/resteasy-netty4-cdi/pom.xml

```diff
@@ -38,8 +38,8 @@
 
         <!-- Test dependencies -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
```

Comment:
If this one does not work with JUnit 5 we should file a JIRA for this and make a comment as to why with the JIRA referenced.